zer0-os/zOS
# feat(message-info-panel): add sent by (from) section

### What does this do?
- adds sent by section as shown below.

### Why are we making this change?
- to render who the message is from on the message info panel.

### How do I test this?
- run tests as usual.
- run ui and check message info on a message you didn't send (you received).

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

<img width="410" alt="Screenshot 2024-06-06 at 17 21 52" src="https://github.com/zer0-os/zOS/assets/39112648/88c471e6-0025-4272-8ddf-80fc36c33501">
